azure-core/tests/test_pipeline.py

```diff
@@ -74,6 +86,41 @@ def on_exception(self, requests, **kwargs):
     with pytest.raises(NotImplementedError):
         pipeline.run(req)
 
+class TestRequestsTransport(unittest.TestCase):
+
+    def test_basic_requests(self):
+
+        conf = Configuration()
+        request = HttpRequest("GET", "https://bing.com")
```

Comment:
Just noting this seems to mean the unit tests cannot be run successfully without an internet connection. In other cases I have seen people mock these requests, e.g. 

https://github.com/Azure/azure-sdk-for-python/blob/f0587587eb8a6dd5f76581b796fa2342d2532634/azure-core/tests/test_credentials.py#L35

Reply:
Will add issue to track this